main.tf

```diff
@@ -152,8 +152,8 @@ resource "azurerm_mssql_server_vulnerability_assessment" "this" {
   storage_account_access_key      = var.storage_account_access_key
 
   recurring_scans {
-    enabled                   = true
-    email_subscription_admins = false
-    emails                    = []
+    enabled                   = var.vulnerability_assessment_recurring_scan_enabled
+    email_subscription_admins = var.vulnerability_assessment_recurring_scan_email_subscription_admins
+    emails                    = var.vulnerability_assessment_recurring_scan_emails
```

Comment:
Rename variables from `*_recurring_scan_*` to `*_recurring_scans_*`:

```suggestion
    enabled                   = var.vulnerability_assessment_recurring_scans_enabled
    email_subscription_admins = var.vulnerability_assessment_recurring_scans_email_subscription_admins
    emails                    = var.vulnerability_assessment_recurring_scans_emails
```

To follow variable naming convention = resource name + block name + argument name.